### PR TITLE
add review comment summary document

### PR DESCRIPTION
**Pull request by chunfuwen**
this doc may provide some kind of checklist
before submitting PR

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
@balamuruhans 
I give one short summary about review issues,please feel free add more to it

File: tp-libvirt_review_comment_summary.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+For beginners or practice executor, it may provides some self-check before submitting to normal review in github. 
+======================================================================
+Goal:
+
+1)Provide one place to accumulate knowledge 
+2)Provide some helps in coding or style format for tp-libvirt to beginners or executors 
+
+======================================================================
+More common issues during reviewing can be categorized as below.
+Fortmat:
```
Comment (sathnaga) [inline tp-libvirt_review_comment_summary.rst]: s/Fortmat/Format

File: tp-libvirt_review_comment_summary.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+======================================================================
+Enhancement(Best practice):
+	-- conjunction two folders:use os.path.join(xx,xx)
+	-- avoid too generic logging message
+	-- not recommended to use mutable default value as an argument see: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html
+	-- without timeout value in infinite loop
+	-- use with to open files
+
+
```
Comment (sathnaga) [inline tp-libvirt_review_comment_summary.rst]: make sure this is run before sending patch,
`inspekt checkall --no-license-check <*>.py`

File: tp-libvirt_review_comment_summary.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Overview:
+
+This doc summarize the most common review comments in tp-libvirt open source project. 
+For beginners or practice executor, it may provides some self-check before submitting to normal review in github. 
+======================================================================
+Goal:
+
+1)Provide one place to accumulate knowledge 
+2)Provide some helps in coding or style format for tp-libvirt to beginners or executors
```
Comment (balamuruhans) [inline tp-libvirt_review_comment_summary.rst]: Good initiative @chunfuwen, this doc helps everyone.

File: tp-libvirt_review_comment_summary.rst (hunk 1 above)
Comment (chunfuwen) [reply tp-libvirt_review_comment_summary.rst]: fixed

File: tp-libvirt_review_comment_summary.rst (hunk 2 above)
Comment (chunfuwen) [reply tp-libvirt_review_comment_summary.rst]: fixed

File: tp-libvirt_review_comment_summary.rst (hunk 3 above)
Comment (chunfuwen) [reply tp-libvirt_review_comment_summary.rst]: Thanks

**Comment by balamuruhans** [comment]
@chunfuwen can we mention a line about this file to refer before submitting PR in README.rst ?

**Comment by chunfuwen** [comment]
@balamuruhans , not very clear about your comment 'can we mention a line about this file to refer before submitting PR in README.rst ?'

**Comment by balamuruhans** [comment]
@chunfuwen In README.rst, let us mention one of the point as adhering to checklist mentioned in review_comment_summary.rst before submitting PR to the repository,

**Example**
```
**README.rst**
----------------
6. Ensure check-list in review_comment_summary.rst are met
7. Fix any problems
8. Push your changes and submit a pull request
9. That's it.
```

**Comment by chunfuwen** [comment]
@balamuruhans ,comment had been addressed,please take a look.

**Pull request merged**